specs/graphite/gt_branch_checkout.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Graphite - Checkout a branch
+command: "gt bco {{branch_name}}"
+tags:
+  - graphite
+description: "Often used when wanting to check out a singular branch for collaboration OR personal use."
```

Comment:
I think it might be useful to the user if we referenced the git version of this command? e.g. when I'm searching for the workflow in Warp and I don't remember what the command is, I'd type "git checkout" 

"Similar to git checkout <branch_name>"?